src/components/RecipeDetails/RecipesNutrition.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import RecipeNutritionItem from "./RecipeNutritionItem";
+
+const ProductNutrition = ({ nutrition }) => {
+  const { ENERC_KCAL, FAT, FIBTG, PROCNT, SUGAR, CA, CHOCDF, CHOLE } =
+    nutrition;
+
+  const items = [ENERC_KCAL, FAT, FIBTG, PROCNT, SUGAR, CA, CHOCDF, CHOLE];
```

Comment:
This feels a bit.. repetitive? Can you maybe come up with a function that does this for you?